src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/RunFunnelByRotations.java

```diff
@@ -0,0 +1,27 @@
+package edu.greenblitz.robotName.commands.shooter.funnel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class RunFunnelByRotations extends FunnelCommand{
+
+    private Rotation2d rotations;
+
+    public RunFunnelByRotations(Rotation2d rotations) {
+        this.rotations = rotations;
+    }
+
+    @Override
+    public void initialize() {
+        funnel.resetEncoder(new Rotation2d(0));
+    }
+
+    @Override
+    public void execute() {
+        funnel.moveToPosition(rotations);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return funnel.isAtAngle(rotations);
+    }
+}
```

Comment:
dont we need an end? where do we stop the funnel

Reply:
the end is in the FunnelCommand

Reply:
The end is in FunnelCommand